components/sysutils/mtx/patches/02-install-man.patch

```diff
@@ -0,0 +1,17 @@
+It is not necessary to install the manpages. IPS finds them in src path
```

Comment:
But `gmake sample-manifest` does not.  It is better when the `sample-manifest` match as close as possible the real manifest, so if the build works properly without this patch then please remove it.